### PR TITLE
sources: add empty file to placate twoliter

### PR DESCRIPTION
**Pull request by jmt-lab**
**Description of changes:**
Add empty sources folder

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by cbgbt — APPROVED** [review]
Such files are nice to add to README.md since GitHub renders those, but not gonna block on that!

**Comment by ginglis13** [comment]
before the force push, CI was failing with

```
/home/runner/work/bottlerocket-kernel-kit/bottlerocket-kernel-kit/build/tools/docker-go --module-path /home/runner/work/bottlerocket-kernel-kit/bottlerocket-kernel-kit/sources/ecs-gpu-init --sdk-image public.ecr.aws/bottlerocket/bottlerocket-sdk:v0.50.0 --go-mod-cache /home/runner/work/bottlerocket-kernel-kit/bottlerocket-kernel-kit/.gomodcache/pkg/mod --command go list -mod=readonly ./... >/dev/null && go mod vendor
pattern ./...: directory prefix . does not contain main module or its selected dependencies
```

So it looks like there's more hackiness going on with `twoliter` and the `sources` dir

https://github.com/bottlerocket-os/bottlerocket-kernel-kit/actions/runs/12302633960/job/34335926119?pr=4

**Review by ginglis13 — APPROVED** [review]
LGTM on green

**Review by cbgbt — APPROVED** [review]

File: sources/deny.toml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+# Deny crates from unknown registries or git repositories.
+unknown-registry = "deny"
+unknown-git = "deny"
+
+allow-git = [
+    # The settings SDK is currently provided as a git dependency,
+    # We will allow it as an exception until the following is resolved:
+    # https://github.com/bottlerocket-os/bottlerocket-settings-sdk/issues/18
+    "https://github.com/bottlerocket-os/bottlerocket-settings-sdk",
+]
```
Comment (bcressey) [inline sources/deny.toml]: Do we need this for some reason?

File: sources/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+This is a temporary workaround for twoliter who currently expects a sources
+directory to exist in order for the docker volume mounts to work in buildkit
```
Comment (bcressey) [inline sources/README.md:2]: Is there a backlog issue in Twoliter to avoid needing this?

Comment (jmt-lab) [reply sources/README.md:2]: I'll be adding it, we had a regression on a bug fix I did.

File: sources/deny.toml (hunk 1 above)
Comment (jmt-lab) [reply sources/deny.toml]: ah no i'll clean it up

**Review by bcressey — APPROVED** [review]

**Review by rpkelly — APPROVED** [review]

**Pull request merged**